lib/config.rb

```diff
@@ -38,6 +38,10 @@ def define_entries
         description: "Enable experimental features. See " \
           "https://github.com/SUSE/machinery/wiki/Experimental-Features for more details"
       )
+      entry("perform_supportability_check",
```

Comment:
I know I suggested it but after reading it here I think that `supportability` does not really make sense. `perform_support_check`?